src/benchmarks/micro/corefx/System.Text.RegularExpressions/Perf.Regex.Cache.cs

```diff
@@ -1,118 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Collections.Generic;
-using System.Threading.Tasks;
-using BenchmarkDotNet.Attributes;
-using MicroBenchmarks;
-
-namespace System.Text.RegularExpressions.Tests
-{
-    [BenchmarkCategory(Categories.CoreFX)]
-    public class Perf_Regex_Cache
```

Comment:
I think we should keep the cache ones, or at least someone that fits that purpose.

Reply:
I reverted the change